src/lib/components/inventory/selected_item_info.ts

```diff
@@ -147,6 +158,22 @@ export class SelectedItemInfo extends FloatElement {
         `;
     }
 
+    renderBluegem(): TemplateResult<1> {
+        if (!this.itemInfo || !this.bluegemData) {
+            return html``;
+        }
+
+        // Some skins got only one blue value
+        if (this.bluegemData.backside_blue === undefined) {
+            return html`<div>Blue: ${this.bluegemData.playside_blue}%</div>`;
+        }
+
+        const placement = this.itemInfo.weapon_type === 'AK-47' ? 'Top / Magazine' : 'Front / Back';
```

Comment:
We perform this check in a couple places -- perhaps the return value of the blue gem data should return the placement string.